docs/portal_config.md

```diff
@@ -388,7 +388,17 @@ Below is an example, with inline comments describing what each JSON block config
     "dropdowns": {} // optional; dropdown groupings for buttons
   },
   "discoveryConfig": { // config for Discovery page. Required if 'featureFlags.discovery' is true. See src/Discovery/DiscoveryConfig.d.ts for Typescript schema.
+    "requireLogin": false, // optional, defaults to false. If true, requires user to sign in before seeing the Discovery page
     "features": {
+      "exportToWorkspaceBETA": { // configures the export to workspace feature. If enabled, the Discovery page data must contain a field which is a list of GUIDs for each study. See `manifestFieldName`
```

Comment:
are we going to call it `BETA`?

Reply:
I was trying to convey that this config is likely to be replaced / updated in a newer version of portal. For HEAL/BDCat rn we're working on a way to configure multiple different export options from the Disco page, I'm imagining that will mean that the config will look different.